### PR TITLE
(PIE-551) Docs Update

### PR DESCRIPTION
**Pull request by coreymbe**
# Summary

Updates to `README.md` to address PIE-551 and potentially PIE-351.

# Detailed Description

Moved **Custom Installation** section to `README.md`. Added a paragraph to point to the Puppet Alert Actions app and repo. Additional updates made to address markdown syntax, remove harmful terminology, PE console updates, and grammatical errors.

# Checklist

[ ] Draft PR?
[x] Ensure README is updated
  [x] Any changes to existing documentation
  [x] Anything new added
  [x] Link to external Puppet documentation
[ ] Tags
[ ] Unit Tests
[ ] Acceptance Tests
[x] PR title is "(Ticket|Maint) Short Description"
[ ] Commit title matches PR title

### CODE REVIEW

**Comment by Magisus** [comment]
Is there a place we could call out what error you might see -- `Could not find terminus splunk_hec for indirection facts` -- if your umask is set wrong?

**Comment by coreymbe** [comment]
@Magisus :: After chatting w/ Bill, there are a few things I need to tweak before this gets merged. So I should be able to find a place to add that error.

**Comment by Magisus** [comment]
I still think it would be good if we could call out the error that happens when `umask` is not set correctly. Otherwise the connection between the (still somewhat buried) recommendation and the error that customers are seeing is not very clear.

File: README.md [hunk 1]
```diff
@@ -35,59 +42,119 @@ There are two Tasks included in this module, `splunk_hec:bolt_apply` and `splunk
...
+
+The steps below will help install and troubleshoot the report processor on a standard Puppet Primary Server; including manual steps to configure compilers (Puppet Servers), and to use the included `splunk_hec` class. Because one is modifying production machines, these steps allow you to validate your settings before deploying the changes live. 
+
+1. Install the Puppet Report Viewer app in Splunk. This will import the needed source types to configure Splunk's HTTP Endpoint Collector (HEC) and provide a dashboard that will show the reports once they are sent to Splunk.
+
+2. Create a Splunk HEC Token or use an existing one that sends to `main` index and **does not** have acknowledgement enabled. Follow the steps provided by Splunk's [Getting Data In Guide](http://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector) if you are new to HTTP Endpoint Collectors.
+
+3. Install this Puppet module in the environment that your Puppet Servers are using (e.g. `production`).
+
+4. Run `puppet plugin download` on your Puppet Servers to sync the content. Some users with strict permissions may need to run `umask 022` first.
```
Comment (bmjen) [inline README.md:103]: I think it would be good to add a note below this step describing the example error that @Magisus mentioned.

**Review by bmjen — CHANGES_REQUESTED** [review]

**Comment by coreymbe** [comment]
@bmjen @Magisus :: Sorry about that, the latest commits were to better organize the previous commits I had. Still plan to add the error to the Custom Installation section. 👍

File: README.md (hunk 1 above)
Comment (coreymbe) [reply README.md:102]: Error messaged and note added.

**Review by Magisus — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,93 +1,165 @@
...
+
+  * **Please Note**: If permissions are too restrictive you may encounter the following error:
```
Comment (Magisus) [inline README.md:104]: Can we add "in the Puppet Server logs" to this?

Comment (coreymbe) [reply README.md]: Just added that bit to that line.

**Review by Magisus — APPROVED** [review]
LGTM!

**Review by bmjen — APPROVED** [review]

**Pull request merged**